python/semantic_kernel/connectors/memory/pinecone/pinecone_memory_store.py

```diff
@@ -81,7 +81,7 @@ async def create_collection(
         self,
         collection_name: str,
         dimension_num: int | None = None,
-        distance_type: str | None = "cosine",
+        distance_type: str | None = "cosine_similarity",
```

Comment:
nit:
```suggestion
        distance_funcion: DistanceFunction | None = DistanceFunction.COSINE_SIMILARITY,
```

Reply:
this is the old memory store, too broad find and replace, reverted.